### PR TITLE
released-label plugin

### LINKED ISSUE #194: "released" label
**Is your feature request related to a problem? Please describe.**

add a `released` label to PRs that have been released

### PR DESCRIPTION
**Pull request by hipstersmoothie**
# What Changed

a plugin to add a released label and comments

# Why

closes #194 

Todo:

- [ ] Add tests
- [x] Add docs
- [ ] Add yourself to contributors (run `yarn contributors:add`)

### CODE REVIEW

**Comment by adierkens** [comment]
:rocket: PR was released in v4.2.0 :rocket:

**Comment by zephraph** [comment]
Dang! 🙌 

So good.

File: docs/pages/released-label.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Released Label Plugin
+
+This plugin
+
+- comments on the merged PR with the new version
+- comments on closed issues with the new version
+- adds a `released` label to the pull request
+
+::: message is-warning
+Make sure that you create the `released` label on you project
```
Comment (zephraph) [inline docs/pages/released-label.md]: Wait, does it still expect you to manually add the label?

Comment (hipstersmoothie) [reply docs/pages/released-label.md]: actually after testing GitHub seems to have created the label. I opened #254 to try to tackle this for `create-labels`

**Pull request merged**